### ISSUE TITLE
Rclone Container

### ISSUE THREAD

**Issue by quorn23** [GhostWriters/DockSTARTer#863]
**Is your feature request related to a problem? Please describe.**

Often the usage around such a setup as possible with DockSTARTer involves having things like google drive mounted with rclone, hence a rclone container as option to install would be swell

**Comment by nemchik** [comment]
I'm not against this. Do you have an example of a container that would work well for this purpose?

**Comment by timelf123** [comment]
Official image: https://hub.docker.com/r/rclone/rclone
Unofficial with healthchecks.io and cron support: https://hub.docker.com/r/pfidr/rclone/

**Comment by nemchik** [comment]
I've thought about it a bit more and I don't think this is a good fit for DS. Rclone can be used in a LOT of different says, and has tons of configuration options. Just for example there are is a whole page just for the various commands https://rclone.org/commands/ that rclone includes. Personally I use `mount`, but some others might want `sync` or `copy` or `move`. Each of those commands has different configurations. rclone can be used to connect to many different storage providers, and each of them have different config options as well.

There's also an infinite way of setting up where the storage should go and how it is used.

Long story short I don't think DS is setup to handle all of how the rclone container could be used.
